### DISCUSSION TITLE
Thank you for making and maintaining `mkdocs-material`

### DISCUSSION THREAD

**Discussion by HonkingGoose** [squidfunk/mkdocs-material#3103] [General]
### Background

I helped to update the Renovate docs to the latest version of `mkdocs-material`. The update went smoothly, even though we needed to update from `4.6.3` to `7.2.6`... 🙈 

### New stuff we got thanks to updating

I really like the new features that we got by updating:

- Dark mode.
- Light and dark mode toggle.
- Media query that respects user preference for light/dark mode.
- Add stats to our header with link to the Renovate GitHub repository.
- Edit button for docs pages that takes you to the correct file on GitHub (in most cases, we're doing some funky auto-generating things on our end).
- Back to top button.
- Improved search styling.

All features are well documented, it was easy to find out how to change things to fit our requirements.
I would never have been able to build any of this stuff myself, so it's nice that you all did the hard work. 😉 

### Thank you for creating/maintaining/contributing to `mkdocs-material`

I know that maintaining a project of any size is a lot of work, and people don't get thanked nearly often enough for their work and time spent on a project.

So I wanted to say thank you to the creators, maintainers, and any contributors of `mkdocs-material`. 😉

**Comment by squidfunk** [comment]
Thanks for those words, they really mean a lot to me! I'm happy to see that this project has seen so widespread adoption, having a solid growth rate lately. I'm also happy to hear that the update went so smoothly, even though you skipped 2 major versions.

I have to note that all features, except for the edit button which was there since version 0.1.0 were only possible because of my [awesome sponsors](https://squidfunk.github.io/mkdocs-material/insiders/#how-to-become-a-sponsor). Without the financial support, I wouldn't have had the time to work on those features, so it's likely that they wouldn't have been available by now. So from me, a big, BIG THANK YOU to my sponsors – I'm trying my best on this project 🙏
